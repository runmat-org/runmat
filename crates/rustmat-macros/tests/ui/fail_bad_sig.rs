use rustmat_macros::runtime_builtin;

#[runtime_builtin(name = "oops")]
fn bad(x: i32) -> i32 {
    x
}

fn main() {} 