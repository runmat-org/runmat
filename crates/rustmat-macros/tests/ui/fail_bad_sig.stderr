error[E0277]: the `?` operator can only be applied to values that implement `Try`
 --> tests/ui/fail_bad_sig.rs:3:1
  |
3 | #[runtime_builtin(name = "oops")]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the `?` operator cannot be applied to type `i32`
  |
  = help: the trait `Try` is not implemented for `i32`
